llama_index/agent/openai_agent.py

```diff
@@ -259,14 +259,19 @@ def _chat(
         n_function_calls = 0
 
         # Loop until no more function calls or max_function_calls is reached
+        current_func = function_call
         while True:
-            llm_chat_kwargs = self._get_llm_chat_kwargs(functions, function_call)
+            llm_chat_kwargs = self._get_llm_chat_kwargs(functions, current_func)
             agent_chat_response = self._get_agent_response(mode=mode, **llm_chat_kwargs)
             if not self._should_continue(self.latest_function_call, n_function_calls):
                 logger.debug("Break: should continue False")
                 break
             assert isinstance(self.latest_function_call, dict)
             self._call_function(tools, self.latest_function_call)
+            # change function call to the default value, if a custom function was given
+            # as an argument (none and auto are predefined by OpenAI)
+            if type(current_func) is not str or current_func.lower() not in ["auto", "none"]:
```

Comment:
I thiiiink you can just reduce this to `if current_func not in ('auto', 'none'):` right?

the `auto` and `none` statements are case-sensitive, so no need to call lower(). And using a tuple rather than a list for `in` is usually preferred 

Sorry, major nitpick haha

Reply:
No, nitpicking is fine. I'm usually work with C# .NET and Java, but not with Python. I fixed, what you requested.